Source/tSQLt.Private_PrepareFakeFunctionOutputTable.ssp.sql

```diff
@@ -0,0 +1,26 @@
+IF OBJECT_ID('tSQLt.Private_PrepareFakeFunctionOutputTable') IS NOT NULL
+    DROP PROCEDURE tSQLt.Private_PrepareFakeFunctionOutputTable;
+GO
+---Build+
+CREATE PROCEDURE tSQLt.Private_PrepareFakeFunctionOutputTable
+    @FakeDataSource NVARCHAR(MAX),
+    @OutputTable    NVARCHAR(MAX) OUTPUT
+AS
+    BEGIN
+        SET @OutputTable = tSQLt.Private::CreateUniqueObjectName();
+
+        IF ( LOWER(LTRIM(@FakeDataSource)) LIKE 'select%'
+             AND OBJECT_ID(@FakeDataSource) IS NULL
+           )
+            BEGIN
+                SET @FakeDataSource = CONCAT(N'(', @FakeDataSource, N') a');
```

Comment:
We are still supporting 2008R2. CONCAT is not available in that version.
([https://docs.microsoft.com/en-us/previous-versions/sql/sql-server-2008-r2/ms181984(v=sql.105)](https://docs.microsoft.com/en-us/previous-versions/sql/sql-server-2008-r2/ms181984(v=sql.105)))